packages/ui/src/shared/yt-types.d.ts

```diff
@@ -234,6 +234,10 @@ export interface SettingsConfig {
 }
 
 export interface ConfigData {
+    userSettingsConfig?: {
```

Comment:
it is enough to pass only cluster:
```ts
userSettingsCluster?: string;
```
`userSettingsConfig.mapNodePath` should be never used from browser.